pandas/io/excel.py

```diff
@@ -1322,6 +1328,14 @@ def write_cells(self, cells, sheet_name=None, startrow=0, startcol=0):
             wks.title = sheet_name
             self.sheets[sheet_name] = wks
 
+        if (
+            freeze_panes is not None and
```

Comment:
maybe make this a validation function

``
freeze_panes = _validate_freeze_panes(freeze_panes)
if freeze_panes:
    ....
```